integration_tests/integration_test.py

```diff
@@ -211,11 +211,12 @@ def test_demultiplexing(self):
                 ),
                 self.cfg['demultiplexing']['lane_qc']
             )
-        self.expect_stage_data(['setup', 'wellduplicates', 'bcl2fastq', 'phixdetection', 'fastqfilter', 'seqtkfqchk',
-                               'md5sum', 'fastqc', 'integritycheck', 'qcoutput1', 'dataoutput', 'cleanup',
-                               'samtoolsdepthmulti', 'picardinsertsizemulti', 'qcoutput2', 'runreview',
-                               'picardmarkduplicatemulti', 'samtoolsstatsmulti', 'bwaalignmulti', 'waitforread2',
-                               'bcl2fastqhalfrun', 'picardgcbias'])
+        self.expect_stage_data([
+            'setup', 'wellduplicates', 'bcl2fastq', 'phixdetection', 'fastqfilter', 'seqtkfqchk', 'md5sum', 'fastqc',
+            'integritycheck', 'qcoutput1', 'dataoutput', 'cleanup', 'samtoolsdepthmulti', 'picardinsertsizemulti',
```

Comment:
Can we name the stages in a more informative way than "qc_output_1"?

Reply:
That would be done in a different PR but we could discuss alternative naming conventions.
However we want the stage name to be short and readable because they appear on the reporting app.